src/fides/api/schemas/property.py

```diff
@@ -4,7 +4,7 @@
 from fides.api.schemas.base_class import FidesSchema
 
 
-class MinimalPrivacyExperience(FidesSchema):
+class MinimalPrivacyExperienceConfig(FidesSchema):
     """
     Minimal representation of a privacy experience, contains enough information
     to select experiences by name in the UI and an ID to link the selections in the database.
```

Comment:
Thank you so much for following up on this 🙏 Could you change the docstring here as well?

Reply:
done!